nss-day-cohort-45/tabloidfullstack-giddy-giraffes
# Rcd post styling

## Changes
1. Added styling to Post Cards on Posts and My Posts pages
2. Fixed but in Post Form where CreateDateTime was being used instead of PublishedDateTime
3. Added ability to add a new post from My Posts page 

## Testing
1. git fetch --all
2. git checkout rcd-post-styling
3. Run Server
4. Run Client
5. Log in 
6. Check out the Posts and My Posts pages. 
7. Hover over Post cards to make sure that they pop up and slightly change color. (Definitely let me know if the color and transition is too much, if the color needs to be changed or everything removed altogether).

8. Add a new post from either page and make sure that the date is correct
